src/components/Datasets/Datasets.js

```diff
@@ -155,7 +155,11 @@ const Datasets = ({
     setPageData(state => {
       return {
         ...state,
-        ...generatePageData(handleRequestOnExpand, !isEveryObjectValueEmpty(selectedItem))
+        ...generatePageData(
+          handleRequestOnExpand,
+          !isEveryObjectValueEmpty(selectedItem),
```

Comment:
remove second argument, and pass just selectedItem, then where you need check if it's empty directly in the function